cocos/animation/marionette/animation-controller.ts

```diff
@@ -151,8 +161,9 @@ export class AnimationController extends Component {
      * @zh 获取动画图实例中下一个状态上添加的所有动画剪辑的运行状况。
      * @en Gets the running status of all the animation clips added on the next state in the animation graph instance.
      * @param layer @en Index of the layer. @zh 层级索引。
-     * @returns @en Iterable to the animation clip statuses on next state. An empty iterable is returned in case of no transition.
-     *          @zh 到下一状态上包含的动画剪辑运作状态的迭代器，若未在进行过渡，则返回一个空的迭代器。
+     * @returns @en Iterable to the animation clip statuses on next state.
+     *              An empty iterable is returned in case of no transition or next state is not a motion state.
+     *          @zh 到下一状态上包含的动画剪辑运作状态的迭代器，若未在进行过渡或下一状态不是动画状态，则返回一个空的迭代器。
      */
     public getNextClipStatuses (layer: number) {
```

Comment:
Here also fix that `getNextClipStatuses` will throw if `getCurrentTransition(layer) === null`